_fellows/pamela-jennings-phd.md

```diff
@@ -12,4 +12,8 @@ tags: ['east','2022',Change_Management,Strategy_And_Growth]
+Pamela L. Jennings, Ph.D., MBA, is a design innovator and change management wrangler with experience in the federal government, higher education, and industry. Detailed at the U.S. Department of Transportation Office of the Secretary of Research (OST-R), Pamela’s projects respond to several federal mandates including the Bipartisan Infrastructure Law, the White House Executive Order on Equity, and the implementation of the U.S. DOT Research, Development, and Technology Strategic Plan.
+
+Pamela’s research, development, and technology projects are grounded by creative inquiry. Her career was anchored at IBM Almaden Research Center and SRI International as a design researcher on human centered computing and education technology research projects. Pamela served as a National Science Foundation Program Director in Computer, Information Science, and Engineering (CISE) Human Centered Computing where she led the NSF CreativeIT program. She has participated as a contributor and lead in several National Academies of Sciences projects positioned to elevate integrative learning and research in higher education. This includes the 2018 consensus report, “Integration of the Humanities and Arts with Sciences, Engineering, and Medicine in Higher Education: Branches from the Same Tree”. Pamela has developed program frameworks that integrate learning, research, creativity, innovation, and economic development. She believes it is the nexus of these platforms where breakthroughs and paradigm shifts are made. In her career she has experienced, the juxtaposition of open-innovation research and development in corporate think tanks; the possibilities of generating creatively applied research at federally funded think tanks; the differences in learning models from the arts to the sciences; how the arts and humanities can generate speculative inquiry that informs STEM research and vice versa; the constrictions of the possible when institutional parochialism is too strong; the creative genius that blossoms when given, space and time in places that embrace and nurture the imagination; the power budgets give to influence the course of critical inquiry, policy, innovation, and change; the ability to greatly increase research and development output and impact with small resource shifts; the complexity of building a space for integrative research and creativity that balances government mandates with grassroot enthusiasm.
```

Comment:
"human-centered"
"Human-Centered"
"National Science Foundation (NSF)"
Tree."
"In her career, she has experienced: 

- the juxtaposition of open-innovation research and development in corporate think tanks. 
- the possibilities of generating creatively applied research at federally funded think tanks. 
- the differences in learning models from the arts to the sciences; how the arts and humanities can generate speculative inquiry that informs STEM research and vice versa. 
- the constrictions of the possible when institutional parochialism is too strong. 
- the creative genius that blossoms when given, space and time in places that embrace and nurture the imagination.
- the power budgets give to influence the course of critical inquiry, policy, innovation, and change.
- the ability to greatly increase research and development output and impact with small resource shifts. 
- the complexity of building a space for integrative research and creativity that balances government mandates with grassroot enthusiasm."